tests/postprocess/test_stats.py

```diff
@@ -0,0 +1,33 @@
+"""Test for simple statistics"""
+import pandas as pd
+import numpy as np
+
+from pytest import warns
+
+from batdata.data import BatteryDataset
+from batdata.postprocess.cycle_stats import CycleTimes
+
+
+def test_times():
+    raw_data = pd.DataFrame({
+        'cycle_number': [0, 0, 1, 1, 2, 2],
+        'test_time': [0, 0.99, 1, 1.99, 2., 2.99]
+    })
+    data = BatteryDataset(raw_data=raw_data)
+    CycleTimes().compute_features(data)
+    assert np.isclose(data.cycle_stats['cycle_start'], [0., 1., 2.]).all()
+    assert np.isclose(data.cycle_stats['cycle_duration'], [1., 1., 0.99]).all()
```

Comment:
Why [1., 1., 0.99] and not simply [0.99] * 3, which is what it should be exactly?

Reply:
The next cycle starts at the earliest timestamp for that cycle, not at the last timestamp from the previous cycle.